index.js

```diff
@@ -29,6 +30,7 @@ class ServerlessAppsyncPlugin {
         + `is no longer supported. See ${MIGRATION_DOCS} for more information`);
     };
     this.hooks = {
+      'before:deploy:initialize': () => this.validateSchema(),
```

Comment:
I frequently deploy my appsync application so I wanted this validation to happen as quickly as possible.